src/mcts/params.cc

```diff
@@ -109,6 +109,15 @@ const OptionId SearchParams::kMultiPvId{
     "multipv", "MultiPV",
     "Number of game play lines (principal variations) to show in UCI info "
     "output."};
+const OptionId SearchParams::kHistoryFillId{
+    "history-fill", "HistoryFill",
+    "When to fill unknown history by repeating the earliest known position."};
+
+FillEmptyHistory SearchParams::GetHistoryFill() const {
```

Comment:
It's better to have `kHistoryFill` of type `FillEmptyHistory` in order not to convert it every time.
To convert string to enum in constructor, some function in an anonymous namespace will work.